server/src/main/java/org/elasticsearch/action/admin/cluster/node/hotthreads/TransportNodesHotThreadsAction.java

```diff
@@ -116,14 +115,12 @@ public static class NodeRequest extends TransportRequest {
 
         NodeRequest(StreamInput in) throws IOException {
             super(in);
-            skipLegacyNodesRequestHeader(TransportVersions.MORE_LIGHTER_NODES_REQUESTS, in);
```

Comment:
Do we want to remove the transport version constant itself?

Reply:
All those constants will be going soon in the v9 upgrade, no need to pull them out one-by-one and risk merge conflicts.